package/src/frontend/media/MediaPool.js

```diff
@@ -52,6 +53,7 @@ export class MediaPool {
         player.playsinline(true); 
       }
       player.updateHooks(hooks || {});
+      player.updateMediaEventsContext(mediaEventsContextData);
```

Comment:
it is mentioned in the commit message already.
* player.updateMediaEventsContext is used to update media context for player on allocation.

Reply:
For me, this line from the commit message re-iterates what is already obvious from the code. Still, I see that it is written in a way that it could be seen to answer some possible 'why' questions.

To be more clear, the specific question that is open for me would be: Why do we need to update a media context, when the context of media in an entry is constant?

Reply:
actually this approach was getting very common in our new setup. that is why I did not consider it that important to reiterate it here again.
but I can write some more words

Reply:
If you think it isn't something that should be in the commit message, you can also write a few quick words here, and in case I still don't understand, we can discuss it tomorrow